packages/web/src/app/web-player/WebPlayer.jsx

```diff
@@ -703,12 +701,23 @@ const WebPlayer = (props) => {
                 isMobile={isMobile}
                 component={TransactionHistoryPage}
               />
-              <Route
-                exact
-                path={ALL_COINS_PAGE}
-                isMobile={isMobile}
-                component={AllCoinsPage}
-              />
+
+              {isArtistCoinsEnabled ? (
```

Comment:
wait PR says we're getting rid of this ff? or are we just moving it to the top-level